### ISSUE TITLE
Product Manager / Owner

### ISSUE THREAD

**Issue by doismellburning** [alphagov/government-service-design-manual#620]
https://www.gov.uk/service-manual/the-team mentions:

> a service manager (the ‘product owner’) probably supported by a product manager

https://www.gov.uk/service-manual/the-team/service-manager.html says:

> However, in many cases it is likely that the service manager won’t have the capacity to be this hands-on, so they are likely to need a dedicated product manager.

and even has a section titled "What’s the difference between a service manager and a product manager?"

Alas https://www.gov.uk/service-manual/the-team is a little light on the details of what a Product Manager _is_.

Indeed https://www.gov.uk/service-manual/the-team/delivery-manager even links to the Service Manager page with the text "Product Manager":

```
They must be able to enable the team to produce estimates of how much effort is required to produce features that the <a href="/service-manual/the-team/service-manager.html">Product Manager</a> wants.
```

So, when a Product Manager is _not_ a Service Manager, what's it like?

I'd love a "Product Manager" page similar to the others linked, with a description, guidance, sample job description etc.

Or am I looking in the wrong direction? Is the modal case that the Product Manager and Service Manager are one and the same? In https://speakerdeck.com/kitcollingwood/being-a-service-manager Kit Collingwood-Richardson recommends they be different, and I can totally see why.
Is it that every different scenario is unique?
I could perhaps attempt to reverse-engineer this myself,
but I suspect someone with more familiarity would be able to do a better job and faster!

TL;DR Would love a "The Team" page about Product Managers as distinct from Service Managers if that makes sense please!

**Comment by 36degrees** [comment]
Thanks for your feedback. We've refreshed our guidance on the roles within teams which can now be found in [What each role does in a service team](https://www.gov.uk/service-manual/the-team/what-each-role-does-in-service-team) – and we differentiate between product and service managers there.

If you still have feedback to provide please use the feedback form at the bottom of the page so it can find its way to the relevant team.

**Comment by doismellburning** [comment]
@36degrees Excellent, thanks

> If you still have feedback to provide please use the feedback form at the bottom of the page so it can find its way to the relevant team.

Is that a "don't use GitHub issues because it won't" then?

**Comment by 36degrees** [comment]
Yeah, for feedback on the content, use the feedback form. The content is no longer stored in the GitHub repo and content designers don't monitor GitHub for issues.
